barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.cpp

```diff
@@ -311,6 +314,9 @@ void ClientIVC::complete_kernel_circuit_logic(ClientCircuit& circuit)
     // subtable is at the top of the final aggregate table since it is the last to be prepended).
     if (is_tail_kernel) {
```

Comment:
Can we add an assert here to check that the subtable is empty prior to these ops?